frontend/src/libs/packages/notification/libs/enums/notification.theme.enum.ts

```diff
@@ -0,0 +1,7 @@
+const theme = {
```

Comment:
```suggestion
const NotificationTheme= {
```
also file naming - notification-theme.enum